clcache.py

```diff
@@ -219,26 +229,19 @@ def getManifestHash(compilerBinary, commandLine, sourceFile):
 
     @staticmethod
     def getIncludesContentHashForFiles(includes):
-        listOfIncludesHashes = []
-        includeMissing = False
+        listOfHashes = []
 
-        for path in sorted(includes.keys()):
+        for path in includes:
             try:
-                fileHash = getFileHash(path)
-                if fileHash != includes[path]:
-                    raise IncludeChangedException()
-                listOfIncludesHashes.append(fileHash)
+                listOfHashes.append(getFileHash(path))
             except FileNotFoundError:
-                includeMissing = True
+                raise IncludeNotFoundException
```

Comment:
Hmm, does this change mean that in case the statistics are counted slightly differently in case some included file has a different hash, and another included does not exist anymore?